drivers/serial/uart_nrf_sw_lpuart.c

```diff
@@ -479,26 +506,39 @@ static void uart_callback(const struct device *uart, struct uart_event *evt,
 		break;
 
 	case UART_RX_BUF_RELEASED:
+		LOG_DBG("Rx buf released");
 		user_callback(dev, evt);
 		break;
 
 	case UART_RX_DISABLED:
+	{
+		bool call_cb;
```

Comment:
This variable is initialized below, but does not seem to be used in any way afterwards.

Reply:
it must have get lost in some upmerge. Wrapped callback call with `if (call_cb)`